src/pages/api/data.js

```diff
@@ -1,8 +1,19 @@
 import { fetchAllNodes } from 'api';
 
 export default async function handler(req, res) {
+  const daysAgo = req.query.length;
+  const date = new Date();
+  const dateRange = date.setDate(date.getDate() - daysAgo);
+  let ISOFormat;
+
+  if (dateRange) {
+    ISOFormat = new Date(dateRange).toISOString();
```

Comment:
ISOFormat?

Reply:
...the [date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) string returned is in ISO Format. I can rename it though @kilemensi